.github/workflows/tests.yml

```diff
@@ -112,10 +117,21 @@ jobs:
         ./.venv/bin/pip freeze
 
     # We set `kernel.yama.ptrace_scope=0` for `attachp` command tests
-    - name: Run tests
+    - name: Run GDB Tests
       run: |
         sudo sysctl -w kernel.yama.ptrace_scope=0
         ./tests.sh -d gdb -g gdb
+  
+    - name: Run DBG Tests on GDB
+      run: |
+        sudo sysctl -w kernel.yama.ptrace_scope=0
+        ./tests.sh -d gdb -g dbg
+    
+    - name: Run DBG Tests on LLDB
+      if: matrix.os != 'ubuntu-22.04'
```

Comment:
Why this? Can we add a comment?

Reply:
I was having trouble with a one-off issue that test in particular in 22.04, and since we don't really support it for LLDB, I figured disabling it was okay. Plus, the docker test for 22.04 is still enabled, and it passes fine.